go/vt/vtgate/planbuilder/testdata/aggr_cases.txt

```diff
@@ -2150,7 +2150,7 @@ Gen4 error: In aggregated query without GROUP BY, expression of SELECT list cont
 # Group by invalid column number (code is duplicated from symab).
 "select id from user group by 1.1"
 "column number is not an int"
-Gen4 error: Expression of SELECT list is not in GROUP BY clause and contains nonaggregated column 'id' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by
+Gen4 error: error parsing column number: 1.1
```

Comment:
Why is this failing? It's a valid (although stupid) query, right?

Reply:
We had an unused if statement to deal with this kind of error in the analyzer's `earlyRewrite`. I thought it would be more efficient to fail early instead of delegating the work to later parts of the planner. V3 fails with the same kind of error

Reply:
The old error is the correct one. `order by 1.1` is perfectly valid and should not fail. the missing `group by` is the real error here